packages/framework-core/src/booster-command-dispatcher.ts

```diff
@@ -30,13 +31,13 @@ export class BoosterCommandDispatcher {
 
     const commandClass = commandMetadata.class
     this.logger.debug('Found the following command:', commandClass.name)
-    const command = new commandClass()
+    const command = commandClass as CommandClassInterface
     Object.assign(command, commandEnvelope.value)
```

Comment:
You are assigning the properties to the command class, instead of the command instance. So I don't think this will work as expected